www/viewer.js

```diff
@@ -69,6 +70,25 @@ function getRowCount() {
 	return ((total / rowHeight) + 1) | 0;
 }
 
+var ebcdic2ascii = [
```

Comment:
I assume this is [US CP37](https://en.wikipedia.org/wiki/Code_page_37) but you should clarify it. There are lots of EBCDICs.